content/developer/addons/addon-info.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Addon Information
+tags:
+- Developers
+- Addons
+- Configuration
+- about.php
+- addon.json
+category: addons
+menu:
+  developer:
+    weight: 10
+    parent: addons
+versioning:
+  added: 2.5
+---
+
+Every Vanilla addons must declare certain information about itself. This includes:
+
+- Addon Metadata
```

Comment:
If these are subsections in this page, make anchor links

Reply:
They are not.